lua/soulbound_abilities/beebarrel.lua

```diff
@@ -0,0 +1,133 @@
+local ABILITY = {}
+
+ABILITY.Name = "Place Bee Barrel"
+ABILITY.Id = "beebarrel"
+ABILITY.Description = "Place a bee barrel that will release bees when it explodes"
+ABILITY.Icon = "vgui/ttt/roles/sbd/abilities/icon_beebarrel.png"
+
+local beebarrel_uses = CreateConVar("ttt_soulbound_beebarrel_uses", "3", FCVAR_REPLICATED, "How many uses of the place bee barrel ability should the Soulbound get. (Set to 0 for unlimited uses)", 0, 10)
+local beebarrel_cooldown = CreateConVar("ttt_soulbound_beebarrel_cooldown", "0", FCVAR_NONE, "How long should the Soulbound have to wait between uses of the place bee barrel ability", 0, 10)
+
+table.insert(ROLE_CONVARS[ROLE_SOULBOUND], {
+    cvar = "ttt_soulbound_beebarrel_uses",
+    type = ROLE_CONVAR_TYPE_NUM,
+    decimal = 0
+})
+table.insert(ROLE_CONVARS[ROLE_SOULBOUND], {
+    cvar = "ttt_soulbound_beebarrel_cooldown",
+    type = ROLE_CONVAR_TYPE_NUM,
+    decimal = 1
+})
+
+if SERVER then
+
+    local function BeebarrelDamage(target, dmginfo)
+        if target:GetClass() == "prop_physics" then
+            
+            local model = target:GetModel()
+            if model == "models/bee_drum/beedrum001_explosive.mdl" and dmginfo:GetDamage() >= 1 then
+                //PrintMessage( HUD_PRINTTALK, "beebarrel" )
+                local pos = target:GetPos()
+                timer.Create("barrelbeesspawn",0.1,3,function()
```

Comment:
If this is the same name from the randomat, we'll need something unique

Reply:
I changed the model name to bee_drum/beedrum002_explosive.mdl and added a convar to change the number of bees per barrel